### ISSUE TITLE
segmentation fault

### ISSUE THREAD

**Issue by mehrdadsk** [alexdobin/STAR#1582]
Hi Alex

I am trying to map a demultiplexed fastq file (Read1 with barcode and UMI, read 2 with sequence). This is from tissue of a rat and the genome size is 2.83 gb. Indexing seems done well. But I get segmentation fault error while I am trying to map it.
The codes I am using is:

./STAR --runMode genomeGenerate --genomeDir STAR_Index/ --genomeFastaFiles genome.fa --sjdbGTFfile genes.gtf --genomeSAsparseD 3 --genomeSAindexNbases 12
java -jar BRBseqTools.jar Trim -f myFile.fastq.gz
mkdir BAM/
./STAR --runMode alignReads --genomeDir STAR_Index/ --outFilterMultimapNmax 1 --readFilesCommand gzcat --outSAMtype BAM Unsorted --outFileNamePrefix BAM/ --readFilesIn myFile.trimmed.fastq.gz

and the error is:
Jun 15 13:20:43 ..... started STAR run
Jun 15 13:20:43 ..... loading genome
Jun 15 13:21:13 ..... started mapping
zsh: segmentation fault  ./STAR --runMode alignReads --genomeDir STAR_Index/ --outFilterMultimapNmax 1

Would you please let me know what can I do for fixing this problem?
Best regards

**Comment by alexdobin** [comment]
Hi Mehrdad,

please try to run with the latest patch (if you are not already using it):
https://github.com/alexdobin/STAR/releases/tag/2.7.10a_alpha_220601
If it still seg-faults, please send me the Log.out file.

Cheers
Alex

**Comment by mehrdadsk** [comment]
Dear Alex

Thank you for the reply. I have downloaded the latest patch. But there are a lot of errors when I am trying to compile STAR on mac. I got this error:

```
clang: error: unsupported option '-fopenmp'
make: *** [SoloFeature_collapseUMI_Graph.o] Error 1
```
Isn't there any pre-compiled STAR for the latest patch? 

Thank you

**Comment by mehrdadsk** [comment]
I am getting this error when I am compiling STAR on mac

re_collapseUMI_Graph.cpp
clang: error: invalid version number in '-mmacosx-version-min=12.4'
In file included from /usr/local/Cellar/gcc/11.3.0_2/include/c++/11/bits/postypes.h:40,
                 from /usr/local/Cellar/gcc/11.3.0_2/include/c++/11/iosfwd:40,
                 from /usr/local/Cellar/gcc/11.3.0_2/include/c++/11/ios:38,
                 from /usr/local/Cellar/gcc/11.3.0_2/include/c++/11/istream:38,
                 from /usr/local/Cellar/gcc/11.3.0_2/include/c++/11/fstream:38,
                 from SoloFeature.h:4,
                 from SoloFeature_collapseUMI_Graph.cpp:1:
/usr/local/Cellar/gcc/11.3.0_2/include/c++/11/cwchar:44:10: fatal error: wchar.h: No such file or directory
   44 | #include <wchar.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [SoloFeature_collapseUMI_Graph.o] Error 1

**Comment by mehrdadsk** [comment]
An update from me:
The issue with compiling has been solved by uninstalling and reinstalling Xcode. :)
